src/NuGet.Clients/NuGet.CommandLine/CommandLineSourceRepositoryProvider.cs

```diff
@@ -11,20 +12,20 @@ namespace NuGet.CommandLine
     public class CommandLineSourceRepositoryProvider : ISourceRepositoryProvider
     {
         private readonly Configuration.IPackageSourceProvider _packageSourceProvider;
-        private readonly List<Lazy<INuGetResourceProvider>> _resourceProviders;
+        private readonly IEnumerable<Lazy<INuGetResourceProvider>> _resourceProviders;
```

Comment:
no hard reason, just exposes lesser API as the constructor defines the list, and it should be treated read-only from then on